styles/frame-styles.css

```diff
@@ -10,12 +10,16 @@
     --corners-radius: 5px;
     --splitter-width: 4px;
 
+    --body-container-padding-left: 4px;
+
     --layout-container-padding-top: 0px;
     --layout-container-padding-bottom: 5px;
-    --layout-container-padding-left: 5px;
```

Comment:
why 4 to 5?

Reply:
just to get the padding/spacing just right